bindings/python/tests/test_read.py

```diff
@@ -49,12 +49,26 @@ def test_sync_reader(service_name, operator, async_operator):
         assert not reader.writable()
         assert not reader.closed
 
-        read_content = reader.read()
+        read_content = bytearray()
```

Comment:
Hi, `reader.read()` has different semantics from `reader.read(size)`. The test here is correct, so please don't change it.

Reply:
got it